test/suite/test_hs11.py

```diff
@@ -76,8 +77,17 @@ def test_key_deletion_clears_hs(self):
             self.session.begin_transaction('read_timestamp=' + timestamp_str(ts))
             for i in range(1, 10000):
                 if i % 2 == 0:
-                    cursor.set_key(str(i))
-                    self.assertEqual(cursor.search(), wiredtiger.WT_NOTFOUND)
+                    if update_type == 'deletion':
+                        cursor.set_key(str(i))
+                        self.assertEqual(cursor.search(), wiredtiger.WT_NOTFOUND)
+                    else:
+                        self.assertEqual(cursor[str(i)], value2)
                 else:
                     self.assertEqual(cursor[str(i)], value1)
             self.session.rollback_transaction()
+
+    def test_key_deletion_clears_hs(self):
+        self.run_test('deletion')
```

Comment:
We normally use `make_scenarios` to do this.

Reply:
Fixed